packages/hub/src/lib/download-file-to-cache-dir.ts

```diff
@@ -100,6 +100,9 @@ export async function downloadFileToCacheDir(
 	await mkdir(dirname(blobPath), { recursive: true });
 	await mkdir(dirname(pointerPath), { recursive: true });
```

Comment:
We might want to check for exists before mkdir the parent directory then, so we would avoid calling mkdir